omnigibson/utils/transform_utils.py

```diff
@@ -697,7 +697,7 @@ def axisangle2quat(vec, eps=1e-6):
     angle = th.norm(vec, dim=-1, keepdim=True)
 
     # Create return array
-    quat = th.zeros(th.prod(th.tensor(input_shape)), 4, device=vec.device)
+    quat = th.zeros(th.prod(th.tensor(input_shape, dtype=th.int)), 4, device=vec.device)
```

Comment:
why are these `dtype` needed?

Reply:
oh it's just to pacify some torch warnings that were being printed about the dtype being inferred. Better to be explicit than implicit, I assume?